### PR TITLE
Add arcam_fmj media player

### PR DESCRIPTION
**Pull request by elupus**
**Description:**
Add documentation for new support of arcam receivers.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24621

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by elupus** [comment]
Argh.. forgot that my lib currently requires python 3.6. Ugh.

File: source/_components/arcam_fmj.markdown [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+The `arcam_fmj` platform allows you to control [Arcam FMJ Receveivers](https://www.arcam.co.uk/range/fmj.htm) from Home Assistant.
+
+Supported devices:
+
+- AVR 380 
+- AVR 450
+- AVR 750
+- Likely other AVRs
+
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown:24]: Remove additional line.
```suggestion
## {% linkable_title Configuration %}
```

File: source/_components/arcam_fmj.markdown [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+description: "Instructions on how to integrate Arcam FMJ Receivers into Home Assistant."
+date: 2019-04-28 13:59 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: arcam.svg
+ha_category: Media Player
+ha_release: 0.96
+redirect_from:
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]: Please remove this. Redirects are not needed... (this page did not exist before).

File: source/_components/arcam_fmj.markdown [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+```yaml
+# Minimal example configuration.yaml entry
+arcam_fmj:
+  - host: hostname
+    zone:
+      1:
+```
+
+```yaml
+# Larger example configuration.yaml entry
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]: Move larger/more extensive examples to the bottom, below the configuration options table.

File: source/_components/arcam_fmj.markdown [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+{% configuration %}
+host:
+  description: IP address or hostname of the device.
+  required: true
+  type: string
+port:
+  description: Port to connect to.
+  required: false
+  default: 50000
+  type: int
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
  type: integer
```

File: source/_components/arcam_fmj.markdown [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        type: string
+        default: Arcam FMJ - ZONE_INDEX
+      turn_on:
+        description: Service to use when turning on device when no connection is established
+        required: false
+        type: action
+{% endconfiguration %}
+
+
+### {% linkable_title Power state %}
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
## {% linkable_title Power state %}

```

File: source/_components/arcam_fmj.markdown [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+Arcam FMJ receivers turn of their network port when in standby, the component will try to
+reconnect to the receiver every 5 seconds. This mean powering on the first zone is not
+possible over the builtin network connection. Two options for complete power control
+exists: IR or Serial gatway.
+
+#### IR command
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
### {% linkable_title IR command %}
```

File: source/_components/arcam_fmj.markdown [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+Use an IR blaster to send command to turn device on using these discrete codes:
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion

Use an IR blaster to send a command to turn the device on using these discrete codes:

```

File: source/_components/arcam_fmj.markdown [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+ - Zone 1: Protocol: NEC1 Device: 16 Function: 123
+ - Zone 2: Protocol: NEC1 Device: 23 Function: 123
+
+#### Serial Port to network gateway
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
### {% linkable_title Serial Port to a network gateway %}
```

File: source/_components/arcam_fmj.markdown [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+        default: Arcam FMJ - ZONE_INDEX
+      turn_on:
+        description: Service to use when turning on device when no connection is established
+        required: false
+        type: action
+{% endconfiguration %}
+
+
+### {% linkable_title Power state %}
+Arcam FMJ receivers turn of their network port when in standby, the component will try to
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
Arcam FMJ receivers turn off their network port when in standby, the component will try to
```

File: source/_components/arcam_fmj.markdown [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+reconnect to the receiver every 5 seconds. This mean powering on the first zone is not
+possible over the builtin network connection. Two options for complete power control
+exists: IR or Serial gatway.
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
exists: IR or Serial gateway.
```

File: source/_components/arcam_fmj.markdown [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+#### IR command
+Use an IR blaster to send command to turn device on using these discrete codes:
+ - Zone 1: Protocol: NEC1 Device: 16 Function: 123
+ - Zone 2: Protocol: NEC1 Device: 23 Function: 123
+
+#### Serial Port to network gateway
+Use a network to serial port gateway to connect to the serial port of the
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
Use a network to a serial port gateway to connect to the serial port of the
```

File: source/_components/arcam_fmj.markdown [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+receiver. The serial port is always available and can power on device.
```
Comment (frenck) [inline source/_components/arcam_fmj.markdown]:
```suggestion
receiver. The serial port is always available and can power on the device.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @elupus! I've reviewed it and left some comments. Could you please take a look? Awesome! 👍

File: source/_components/arcam_fmj.markdown (hunk 5 above)
Comment (elupus) [reply source/_components/arcam_fmj.markdown]: was the extra blank line intentional?

File: source/_components/arcam_fmj.markdown (hunk 7 above)
Comment (elupus) [reply source/_components/arcam_fmj.markdown]: Intentional extra lines? If so should all titles have blank lines after?

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/arcam_fmj.markdown [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+The `arcam_fmj` integration allows you to control [Arcam FMJ Receveivers](https://www.arcam.co.uk/range/fmj.htm) from Home Assistant.
+
+Supported devices:
+
+- AVR 380 
+- AVR 450
+- AVR 750
+- Likely other AVRs
+
+## {% linkable_title Configuration %}
```
Comment (MartinHjelmare) [inline source/_components/arcam_fmj.markdown]: We shouldn't use `linkable title` anymore, correct @frenck? Just `## Configuration`

Comment (klaasnicolaas) [reply source/_components/arcam_fmj.markdown]: Thats right, it's no longer needed. But it doesn't break anything if you let it in, it continues to work 😉

Comment (MartinHjelmare) [reply source/_components/arcam_fmj.markdown]: Thanks for clarification! :+1:

Comment (elupus) [reply source/_components/arcam_fmj.markdown]: I'll correct the pull before merge. I still have a day or two :)

File: source/_components/arcam_fmj.markdown [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+description: "Instructions on how to integrate Arcam FMJ Receivers into Home Assistant."
+date: 2019-04-28 13:59 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: arcam.svg
+ha_category: Media Player
+ha_release: 0.96
+---
```
Comment (MartinHjelmare) [inline source/_components/arcam_fmj.markdown:14]: We should add `ha_iot_class`.

Comment (frenck) [reply source/_components/arcam_fmj.markdown:14]: Added

**Review by frenck — APPROVED** [review]

**Pull request merged**